python/dgl/graphbolt/base.py

```diff
@@ -164,6 +164,14 @@ class CSCFormatBase:
     indptr: torch.Tensor = None
     indices: torch.Tensor = None
 
+    def __init__(self, indptr: torch.Tensor, indices: torch.Tensor):
+        self.indptr = indptr
+        self.indices = indices
+        if not indptr.is_cuda:
+            assert self.indptr[-1] == len(
```

Comment:
Is that means tensor access via `[]` is pretty time-consuming, so you are trying to avoid it in the code?

Reply:
Yes, see #6862.

Reply:
I am showing the profile there with pictures.